golem-worker-executor-base/src/services/golem_config.rs

```diff
@@ -259,6 +259,7 @@ pub struct S3BlobStorageConfig {
     pub custom_data_bucket: String,
     pub oplog_payload_bucket: String,
     pub compressed_oplog_buckets: Vec<String>,
+    pub minio: bool,
```

Comment:
i think more transparent solution will be provide optional credentials, as @noise64 mentioned

Reply:
it should :) did not want to add static credential options because we don't need them right now and I didn't like adding that particular option and not all the other possible ones